actions.yaml

```diff
@@ -43,3 +43,16 @@ sync-resources:
         Space separated list of kubernetes resource types
         to use a filter during the sync. This helps limit
         which missing resources are applied.
+delete-storage-class:
+  description: |
+    Delete storage class which may be in conflict with the underlying cluster.
+  params:
+    name:
+      type: string
+      description: Name of a specific storage class to delete.
+      enum:
+        - cephfs
+        - ceph-xfs
+        - cephfs-ext4
```

Comment:
i love that juju supports json-schema for action properties!!

Reply:
look how cool:

```bash
⚡ juju run ceph-csi/leader delete-storage-class name=thing

Operation 1 failed to schedule any tasks:
validation failed: (root).name : must match one of the enum values ["cephfs","ceph-xfs","cephfs-ext4"], given "thing"
```

Reply:
🤩 Amazing stuff!